variants/t1000-e/T1000eBoard.h

```diff
@@ -87,6 +88,9 @@ class T1000eBoard : public mesh::MainBoard {
     #ifdef LED_PIN
     digitalWrite(LED_PIN, LOW);
     #endif
+
+    radio.sleep();
```

Comment:
On my platformio setup, it won't compile if I let this ... don't know about radio on board